src/core/Akka/Dispatch/IRequiresMessageQueue.cs

```diff
@@ -5,8 +5,16 @@
 // </copyright>
 //-----------------------------------------------------------------------
 
+using Akka.Actor;
+using Akka.Dispatch.MessageQueues;
+
 namespace Akka.Dispatch
 {
+    /// <summary>
+    /// Used to help give hints to the <see cref="ActorSystem"/> as to what types of <see cref="IMessageQueue"/> this
+    /// actor requires. Used mostly for system actors.
+    /// </summary>
```

Comment:
Decided to take mercy on @sean-gilliam and add XML-DOC comments wherever I was spending a lot of time :p